library/simulink-library/MomentumVelocityControl/src/centroidalConversion.m

```diff
@@ -0,0 +1,44 @@
+function centroidalDyn = centroidalConversion(DYNAMICS,FORKINEMATICS,STATE)
```

Comment:
these centroidal -related functions could be moved in the `wbc` library folder. But for now it is ok to have them in the sources of the velocity controller, being the only user of the transformation.